cmd/ef_tests/blockchain/test_runner.rs

```diff
@@ -32,8 +32,9 @@ pub fn run_ef_test(test_key: &str, test: &TestUnit) {
         let block: &CoreBlock = &block_fixture.block().unwrap().clone().into();
         let hash = block.hash();
 
+        let blockchain = Blockchain::default();
```

Comment:
Once you move `store`, you won't be able to use `default`. Maybe the default should be on the evm.